packages/client/components/AdhocTeamMultiSelect/AdhocTeamMultiSelect.tsx

```diff
@@ -101,57 +102,92 @@ export const AdhocTeamMultiSelect = (props: Props) => {
+        const label = typeof option === 'string' ? option : option.label
+        return label.toLowerCase().includes(params.inputValue.toLowerCase())
+      })
+
+      const autocompletedEmail = autocompleteEmail(params.inputValue, viewerDomain)
+      const suggestedInvitationOption = createCustomOption(autocompletedEmail)
+
+      if (params.inputValue) {
+        filtered.unshift(suggestedInvitationOption)
+      }
+
+      return filtered
+    }
+  })
 
-        return filtered
-      }}
-      renderTags={(value, getTagProps) =>
-        value.map((option: Option, index: number) => {
-          return (
+  return (
+    <div>
+      <div {...getRootProps()}>
+        <div
+          ref={setAnchorEl}
+          className='align-center flex min-h-[44px] w-full flex-wrap rounded border-2 border-slate-300 bg-white px-1 py-0.5 text-sm font-semibold'
+        >
+          {value.map((option, index: number) => (
             <Chip
               label={option.label}
               picture={option.picture}
               icon={!option.id ? <SendIcon className='text-base' /> : null}
               {...getTagProps({index})}
               key={index}
+              className='m-0.5'
             />
-          )
-        })
-      }
-      renderOption={(props, option) => {
-        return (
-          <li {...props}>
-            {!option.id && <SendIcon className='mr-2 text-base' />}
-            {option.label}
-          </li>
-        )
-      }}
-      renderInput={(params) => (
-        <TextField {...params} variant='filled' placeholder='ex. Traci or traci@company.com' />
-      )}
-    />
+          ))}
+          <input
+            {...getInputProps()}
+            placeholder={!value.length ? 'ex. Traci or traci@company.com' : ''}
+            className='m-0 box-border min-h-[36px] w-0 min-w-[30px] flex-grow border-0 bg-white pl-1 text-black outline-none'
+          />
+        </div>
+      </div>
+      {groupedOptions.length > 0 ? (
+        <ul
+          {...getListboxProps()}
+          className='absolute z-50 mt-0.5 h-auto max-h-64 w-[300px] list-none overflow-y-auto rounded bg-white p-0 shadow-card-1'
+        >
+          {(groupedOptions as Option[]).map((option, index) => {
+            const optionProps = getOptionProps({option, index})
+            const isSelected = optionProps['aria-selected']
+            return (
+              <li
+                key={index}
+                {...optionProps}
+                className={`[&.${autocompleteClasses.focused}]:bg-slate-100 flex h-10 w-full cursor-pointer select-none items-center justify-between rounded px-3 text-sm outline-none hover:bg-slate-100 focus:bg-slate-100 data-[disabled]:pointer-events-none data-[disabled]:opacity-50`}
+              >
+                {!option.id && <SendIcon className='mr-2 text-base' />}
+                <span className={'flex-grow'}>{option.label}</span>
+
+                {isSelected && <CheckIcon className='h-5 w-5' />}
```

Comment:
Rather than show a checkmark when the user is selected, I think we should just show the onHover background colour the whole time:

<img width="346" alt="Screenshot 2023-08-22 at 16 39 26" src="https://github.com/ParabolInc/parabol/assets/39854876/db4fec55-27e8-4f80-aaf7-e3266e042764">